docs/content/configuration/production-cluster.md

```diff
@@ -4,21 +4,40 @@ layout: doc_page
 Production Cluster Configuration
 ================================
 
-__This configuration is an example of what a production cluster could look like. Many other hardware combinations are possible! Cheaper hardware is absolutely possible.__
+```note-info
+This configuration is an example of what a production cluster could look like. Many other hardware combinations are 
+possible! Cheaper hardware is absolutely possible.
+```
 
-This production Druid cluster assumes that metadata storage and Zookeeper are already set up. The deep storage that is used for examples is S3 and memcached is used as a distributed cache.
+This production Druid cluster assumes that metadata storage and Zookeeper are already set up. The deep storage that is 
+used for examples is [S3](https://aws.amazon.com/s3/) and [memcached](http://memcached.org/) is used for a distributed cache.
 
-The nodes that respond to queries (Historical, Broker, and Middle manager nodes) will use as many cores as are available, depending on usage, so it is best to keep these on dedicated machines. The upper limit of effectively utilized cores is not well characterized yet and would depend on types of queries, query load, and the schema. Historical daemons should have a heap a size of at least 1GB per core for normal usage, but could be squeezed into a smaller heap for testing. Since in-memory caching is essential for good performance, even more RAM is better. Broker nodes will use RAM for caching, so they do more than just route queries. SSDs are highly recommended for Historical nodes not all data is loaded in available memory.
+```note-info
+The nodes in this example do not need to be on their own individual servers. Overlord and Coordinator nodes should be 
```

Comment:
To me this line gives an impression that they "have" to be on same machine but theoretically they can be on separate machines...anyways I am OK with it, there is no need to change it.